assemblyline/odm/models/ontology/types/antivirus.py

```diff
@@ -0,0 +1,25 @@
+from assemblyline import odm
+
+
+class Antivirus(odm.Model):
+    class Detection(odm.Model):
+        class Engine(odm.Model):
+            class Definition(odm.Model):
+                update_time = odm.Optional(odm.Date())                          # Time of last update
```

Comment:
I don't think we have the update, also I'm not sure its useful for the ontology. I think the definition field is sufficient.

Reply:
For our ICAP solution in the AntiVirus service, neither definition update_time or definition version are available to us. BUT for other services like MetaDefender, the definition update time and version were both available. My question is, what is the goal of the ontology? Is it to provide a structure for all (useful?) data that a service can generate?

Reply:
Goal is to have a focus on analyst / analytical needs. The update_time I feel gets into more of the debugging space.